make/tests

```diff
@@ -51,10 +51,10 @@ test/%_test.d : test/%_test.cpp
 ##
 
 test/unit/multiple_translation_units%.o : test/unit/multiple_translation_units%.cpp
-	$(COMPILE.c) -O$O $< $(OUTPUT_OPTION)
+	$(COMPILE.cc) -O$O $< $(OUTPUT_OPTION)
 
 test/unit/libmultiple.so : test/unit/multiple_translation_units1.o test/unit/multiple_translation_units2.o
-	$(CC) -shared -fPIC $(OUTPUT_OPTION) $^
+	$(COMPILE.cc) -shared -fPIC $(OUTPUT_OPTION) $^
```

Comment:
for this target, we should be able to just add to the linker options for this target